### ISSUE TITLE
Tinker does not show messages with only a number

### ISSUE THREAD

**Issue by johntiror** [botman/botman#527]
- BotMan Version: 2.0.2
- PHP Version: 7.1.1
- Messaging Service(s): tinker
- Cache Driver: redis

### Description:

If the bot say a number, tinker will not show the number (this does not happen with other divers I testes: messenger and telegram).

### Steps To Reproduce:

This does not work:

```
$botman->hears('number', function (BotMan $bot) {
    $bot->reply(34);
});
```

This works:
```
$botman->hears('number', function (BotMan $bot) {
    $bot->reply('' . 34);
});
```

**Comment by christophrumpel** [comment]
Hey @johntiror , this is due to this line https://github.com/botman/botman/blob/master/src/BotMan.php#L563 (general behaviour) and due to this https://github.com/botman/driver-web/blob/master/src/WebDriver.php#L122-L131 (WebDriver behaviour)

I do not see it as an bug, but it should be consistent right. I dont think it should work with numbers because strings is what we want. Strings our message objects. @mpociot maybe we can parse it to a string if it is an integer?

**Comment by mpociot** [comment]
@christophrumpel I thought about that too. 
Maybe we could just cast the message to string?

**Comment by johntiror** [comment]
I'd cast the message to string since it'll always be a string

**Comment by christophrumpel** [comment]
Yeah, so if it is no instance of Webaccess and no instance of outgoing message, we could cast it to a string, but we would still need to send a Message object then, because in the webdriver we only use the method `getText`. Right?
